core/modules/modulesconf.py

```diff
@@ -0,0 +1,46 @@
+from viur.core.bones import StringBone
+from viur.core.tasks import StartupTask
+from viur.core import conf, db
+from viur.core.i18n import translate
+from viur.core.skeleton import Skeleton, SkeletonInstance
+from viur.core.prototypes import List, BasicApplication
+
+
+class ModulesConfSkel(Skeleton):
+    kindName = "viur-modules-conf"
+    name = StringBone(descr=translate("modulename"), readOnly=True)
+    help_text = StringBone(descr=translate("module helptext"))
+    help_text_add = StringBone(descr=translate("add helptext"))
+    help_text_edit = StringBone(descr=translate("edit helptext"))
+    # seo.....
+
+
+class ModulesConf(List):
+    """
+        This module is for viur internal purposes only.
+        It lists all other modules to be able to provide them with help texts.
+    """
+    kindName = "viur-modules-conf"
+    accessRights = ["manage"]
```

Comment:
Why is this not just _edit_, as in the prototype?
```suggestion
    accessRights = ["edit"]
```